cargo-concordium/src/templates.rs

```diff
@@ -0,0 +1,1179 @@
+#[rustfmt::skip]
```

Comment:
I think it would be better  to have these as separate files. You can then use the https://doc.rust-lang.org/std/macro.include_bytes.html macro to embed it into the binary.

I think that makes it easier to maintain.